composer.json

```diff
@@ -69,6 +69,7 @@
         "doctrine/doctrine-bundle": ">=3",
         "sonata-project/core-bundle": "<3.20",
         "sonata-project/doctrine-extensions": "<1.8",
+        "sonata-project/doctrine-orm-admin-bundle": "<3.32",
```

Comment:
adding a conflict here does not seem right, is it needed to make it work?

Reply:
Versions previous to sonata-project/doctrine-orm-admin-bundle 3.32 haven't the fix.
Actually, the fix is provided by that package.

Reply:
Then, to get the fix you just have to update that package, not this one, right? I mean, we do not conflict on the orm-bundle when we fix something here

Reply:
It's fine, no problem.

Reply:
The commit message and the PR description were also updated.